.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: ['1.8', '9', '11', '12']
```

Comment:
13 is the current version. I don’t think it’s worth testing 9 or 12 (only 8 and 11 are “supported”)